infra/discovery/src/lib/scoring.js

```diff
@@ -0,0 +1,56 @@
+async function scoreListing(listing) {
```

Comment:
Does this need to be async ? I only see synchronous calls in the implementation below ?

Reply:
In the future this may make a DB call, or check some kind of external fraud service, etc...